site/src/docs/components/accordion/index.mdx

```diff
@@ -25,6 +25,8 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 
 - Use accordions to allow the user to quickly glance at the information and then choose to open sections that are interesting to them.
 - **Accordions must be given a header that describes the accordion content.** This header also acts as the label for the expand button. The header level can be chosen depending on the structure of the page.
+  - The `headingLevel` given to the Accordion should be higher than the header in the page content preceeding it (if there is one). For example if the Accordion is nested inside content with a header `h2` then the `headingLevel` for the Accordion should be 3.
+  - Also the Accordion size variant should be chosen so that it nicely fits the content, not putting a really large sized Accordion deep in the page structure.
```

Comment:
I don't think that accordion size information fits here under the header part. The wording also feels "off" from the other sentences. How about putting it matter of factly: "Use the `size` property to change Accordion's size to match your content and rest of the page." ? Or maybe even remove this altogether.